CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 # Change Log (Release Notes)
 
 All _notable_ changes to this project will be documented in this file (`CHANGELOG.md`).
-This project adheres to [the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme) (or at least we try).
+
+This project adheres to [Semantic Versioning 2](http://semver.org/spec/v2.0.0.html), the BigchainDB **public API**:
+* Includes:
+  * This definition of **public API**.
+  * The Command Line Interface.
+  * The transaction schema.
+  * All the endpoints exposed by the HTTP API (including payloads sent to and received from them).
+  * The docker setup.
+* Excludes:
```

Comment:
I agree with this. In the future we may move some of these things to the `Includes` section.

I like the notion of `experimental` since we can have our users testing some features before we make them permanent. We should however have a better definition of  what an `experimental` feature is:
- How to we mark a feature as experimental
- We need to make sure that an experimental feature if in a stable version does not break the server